_site/stylesheet/style.css

```diff
@@ -142,6 +142,36 @@ main .section.section--dark {
 .title__description {
   text-align: center;
 }
+.notes_head{
+  background-color: #03a9f4;
+  color: #fff;
+  font-family: 'Raleway', sans-serif;
```

Comment:
New fonts are not required here, will fallback to default font anyway, better remove this.